src/pages/vysledky.tsx

```diff
@@ -114,6 +121,58 @@ const Vysledky: Page<Partial<TaxFormUserInput>> = ({
     },
   ]
 
+  if (Number(summary.danNaUhradu) > VRCHNA_ZADZBA_PRE_PREDDAVKY && Number(summary.zaplatenePreddavky) == 0) {
+    return (
+      <>
+        <BackLink href={previousRoute} />
+        <h1 className="govuk-heading-l govuk-!-margin-top-3">
+          {`Výpočet dane za rok ${TAX_YEAR}`}
+        </h1>
+        <h2 className="govuk-heading-m govuk-!-margin-top-3">Stručný prehľad</h2>
+        <Summary rows={summaryRows} />
+        <Warning>
+          <strong>
+          Predpokladané mesačné preddavky na rok {TAX_YEAR+1} budú {countPreddavky(taxForm)}€ (výpočet má informatívny charakter). Pre viac informácií navštív web <a href="https://www.financnasprava.sk/sk/elektronicke-sluzby/verejne-sluzby/danove-kalkulacky/vypocet-preddavkov-fo-2023">Finančnej správy</a>.
+          </strong>
+      </Warning>
+        <Link href={nextRoute} legacyBehavior>
+          <button
+            data-test="next"
+            className="govuk-button govuk-!-margin-top-3"
+            type="submit"
+          >
+            Pokračovať
+          </button>
+        </Link>
+      </>
+    )
+  } else if (Number(summary.danNaUhradu) > SPODNA_ZADZBA_PRE_PREDDAVKY && Number(summary.zaplatenePreddavky) == 0) {
```

Comment:
if okolo celej stranky nie je najlepsi, na toto pushnem ja zmenu

Reply:
Ok, ja pozeram na tie testy. Ale musim si trosku osviezit pamat, uz dlho som nenapisal ziadny cypress test.

Reply:
@Ormi pozri zmeny v tomto commite, ono sa to dalo napisat aj takto jednoduchsie 
1ba777141d4b66037086a3767bbbdb3faf575fbb

Reply:
Nice, omnoho elegantnejsie :+1: